arg_scripts/routing_args.py

```diff
@@ -9,6 +9,8 @@ def __init__(self):
         self.paths_matrix = []  # Matrix of potential paths for a single request
         self.mod_formats_matrix = []  # Modulation formats corresponding to each path in paths_matrix
         self.weights_list = []  # Keeping track of one weight of the path (Length, XT, Etc.)
+        self.connection_index = []
+        self.path_index = []
```

Comment:
Comment due to coding guidelines and standards document, same as the comment for `self.connection_index`

Reply:
fixed